resources/web/wwi/nodes/utils/WbFieldChecker.js

```diff
@@ -0,0 +1,38 @@
+function resetIfNonPositive(value, defaultValue) {
+  if (value <= 0) {
+    console.error('Invalid ' + value + ' changed to ' + defaultValue + '. The value should be positive.');
```

Comment:
I would display a warning here, not an error, as it is not fatal.
```suggestion
    console.warn('Invalid ' + value + ' changed to ' + defaultValue + '. The value should be positive.');
```
The same applies in similar functions below.